### PR TITLE
Added npm fmt CI step

### PR DESCRIPTION
**Pull request by Euphoride**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "fmt": "prettier --check ."
```
Comment (aDotInTheVoid) [inline frontend/package.json]: NIT: I'd love this to be called `fmt-check`, to be clear it won't format

File: cmd/groundstation/main.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package main
```
Comment (aDotInTheVoid) [inline cmd/groundstation/main.go]: Why are we bringing this back? It was intentionally deleted

File: frontend/package.json (hunk 1 above)
Comment (Euphoride) [reply frontend/package.json]: Yes, I think that's ideal too

File: cmd/groundstation/main.go (hunk 2 above)
Comment (Euphoride) [reply cmd/groundstation/main.go]: I think switching from an old branch caused some issues, I'll be fixing this soon. Should have labelled the PR draft :P

**Comment by aDotInTheVoid** [comment]
r=me with commits squashed (if you know how to do that, otherwise as-is is fine)

**Pull request merged**